Create Phone Number/index.js

```diff
@@ -0,0 +1,11 @@
+function toString(numbers) {
+  return numbers.join("");
+}
+
+function createPhoneNumber(numbers) {
+  const areaCode = toString(numbers.slice(0, 3));
+  const prefix = toString(numbers.slice(3, 6));
+  const lineNumber = toString(numbers.slice(6));
+
```

Comment:
You don't  need to create another function only to handle toString, you can combine them in one function
```suggestion
function createPhoneNumber(numbers) {
 const areaCode = numbers.slice(0, 3).join("");
  const prefix = numbers.slice(3, 6).join("");
  const lineNumber = numbers.slice(6).join("");


```

Reply:
Hi @mutesialine, I think it's best to avoid repeating yourself when writing code. That's why I created a function to help me convert an array of numbers to a string.